README.md

```diff
@@ -696,6 +696,10 @@ Other options:
   - `eval` — mangle names visible in scopes where eval or with are used
   (disabled by default).
 
+- `keep_fnames` -- default `false`.  Pass `true` when you'd not like to mangle
```

Comment:
Some formatting is wrong here with respect to `eval` above. Perhaps this line must be indented by the same level as `except` and `toplevel` above. `eval` might have to be shifted left by one char.